components/PreviewFrame/index.js

```diff
@@ -26,19 +26,28 @@ const screenSizes = {
   }
 }
 
-const PreviewFrame = ({ previewScreenSize, commitId, repoId, darkmode }) => {
+const PreviewFrame = ({
+  previewScreenSize,
+  commitId,
+  repoId,
+  darkmode,
+  sideBarWidth
+}) => {
   const [scaleFactor, setScaleFactor] = useState(1)
   const [leftSpace, setLeftSpace] = useState(0)
+  const [iframeLoading, setIframeLoading] = useState(true)
   const [colorScheme] = useColorContext()
   const iframeRef = useRef()
 
   const URL = `${PUBLIC_BASE_URL}/repo/${repoId}/preview?commitId=${commitId}&darkmode=${darkmode}`
+  const sideBarWidthConstant = sideBarWidth || SIDEBAR_WIDTH
```

Comment:
maybe I find the use of `constant` confusing here. How about `currentSideBarWidth` or similar?

Reply:
aight